### PR TITLE
Recent installs section

### LINKED ISSUE #708: Frontend: Card in "usage" section of activity dashboard

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

Closes #708
Depends on #728

Implements the recent installs section for the activity dashboard. The 

## Demo

image

### CODE REVIEW

File: frontend/src/components/ActivityDashboard/RecentInstalls.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const installsFormatter = useMemo(
+    () =>
+      new Intl.NumberFormat(i18n.language, {
+        notation: 'compact',
+        maximumFractionDigits: 1,
+      }),
+    [i18n.language],
+  );
+  const formattedInstalls = useMemo(
+    () => installsFormatter.format(pluginRecentStats?.installInLast30Days ?? 0),
```
Comment (manasaV3) [inline frontend/src/components/ActivityDashboard/RecentInstalls.tsx:25]: Based on one of the feedbacks, I have updated this field to be `installsInLast30Days`.

**Review by klai95 — APPROVED** [review]
Looks good to me!

**Review by richaagarwal — APPROVED** [review]
LGTM!

**Review by manasaV3 — APPROVED** [review]

File: frontend/src/utils/HubAPIClient.ts [hunk 2]
```diff
@@ -106,6 +110,16 @@ class HubAPIClient {
 
     return data;
   }
+
+  async getPluginRecentInstallStats(
+    name: string,
+  ): Promise<PluginRecentInstallStats> {
+    const { data } = await this.api.get<PluginRecentInstallStats>(
+      `/activity/${name}/recentStats`,
```
Comment (manasaV3) [inline frontend/src/utils/HubAPIClient.ts:118]: This endpoint should be recent_stats

**Pull request merged**